src/__tests__/integration/data/kv-storage.js

```diff
@@ -0,0 +1,75 @@
+export async function handle(state, action) {
+  const input = action.input;
+  const caller = action.caller;
+
+  if (!state.kvOps) {
+    state.kvOps = {};
+  }
+
+  if (!state.initialized) {
+    state.initialized = true;
+  }
+
+
+  if (input.function === 'mint') {
+    console.log('mint');
+    SmartWeave.kv.put(input.target, input.qty.toString());
```

Comment:
Example put into KV storage

Reply:
The put only adds the 'put' operation (https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/trie/src/types.ts#L74) to the batch operations array.
Changes on the db itself are done only on commit.